src/containers/ForgotPassword.jsx

```diff
@@ -37,52 +38,47 @@ class ForgotPassword extends Component {
     } = this.props;
     const { open } = this.state;
     return (
-      <div className="container">
-        <div className="subContainer">
-          <Scenic />
-        </div>
-        <div className="subContainer">
-          <div className="containerPadding">
-            <h1>
-              Forgot Password
-            </h1>
-            <form onSubmit={handleSubmit(this.onSubmit.bind(this))}>
-              <Field
-                label="Email"
-                name="email"
-                type="text"
-                component={TextInput}
-              />
+      <PageContainer>
+        <div className="containerPadding">
+          <h1>
+            Forgot Password
```

Comment:
could use more whitespace underneath `forgot password`

![image](https://user-images.githubusercontent.com/7741746/68449840-98024400-019d-11ea-8180-bfe4f3aa34fd.png)

Reply:
The design of the application system is in progress. Waiting on design team for mockups. Basically creating temporary styles.